spec/system/candidate_interface/candidate_signing_out_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'A candidate signing out' do
+  context 'when a candidate is signed in' do
+    before do
+      candidate = FactoryBot.create(:candidate)
+      login_as(candidate)
+
+      visit candidate_interface_welcome_path
+    end
+
+    it 'can see a sign out button' do
+      expect(page).to have_selector :link_or_button, 'Sign out'
+    end
+
+    context 'when the sign out button is clicked' do
+      it 'sends the candidate to the start page' do
+        click_link 'Sign out'
+
+        expect(page).to have_current_path(candidate_interface_start_path)
+      end
+
+      it 'can only access start page' do
+        click_link 'Sign out'
+
+        visit candidate_interface_welcome_path
+
+        expect(page).to have_current_path(candidate_interface_start_path)
+      end
+    end
+  end
+
+  context 'when a candidate is not signed in' do
+    it 'does not display a sign out button' do
```

Comment:
not sure, this test is almost identical to the previous one.

Reply:
It's to test that the sign in link is only displayed when the candidate is signed in.
We did TDD this so this may not need to still be here, 
but I think it acts as a nice regression test to make sure the sign out link is being displayed correctly.